### ISSUE TITLE
Colors in 3.0.0-beta.17

### ISSUE THREAD

**Issue by prencher** [s-panferov/awesome-typescript-loader#318]
This is a minor thing, but in 3.0.0-beta.17 (perhaps earlier), the colors were changed around, making them extremely hard to read on Windows:

![screenshot 2016-12-14 10 31 29](https://cloud.githubusercontent.com/assets/138147/21195507/e7d01d4c-c1e8-11e6-8eaa-a1827f6bb518.png)

The actual errors here are an almost unreadable maroon red, while the meta information is bright red. I believe it maps to `gutil.colors.red` for the maroon and `gutil.colors.bold.red` for the bright red, respectively.

Again, minor thing, but would be nice to have colors that work well with default consoles on all platforms.

**Comment by mvestergaard** [comment]
Try out https://conemu.github.io/ which supports ANSI colors, so you aren't limited to 16 colors.

**Comment by prencher** [comment]
Yes, there are plenty of alternatives, all of which is irrelevant to what I was asking for: Reasonable defaults for default terminals that most people use.

**Comment by tomitrescak** [comment]
@prencher as far as I'm concerned this is not a paid help desk, you need to tune it down a bit or do a PR of your own and see if community agrees with your approach. 

Because of post like this OSS developers tend to say ... screw that, not worth it. Long story short, PR for this should be straightforward.
